manifests/facts.pp

```diff
@@ -74,19 +74,21 @@
       command => "'${rubypath}' '${scriptpath}' -o '${factspath}' ${factspid} &> /dev/null",
       minute  => $cron_minutes
     }
-    systemd::timer { "mcollective-facts-refresh.timer":
-      ensure          => $systemd_ensure,
-      active          => $systemd_active,
-      enable          => $systemd_enable,
-      timer_content   => epp("mcollective/refresh_facts.timer.epp", {
+    if $facts["systemd"] == "systemd" {
```

Comment:
@smortex FYI this is wrong 

```
$ sudo facter -p systemd
true
```

This fact is a boolean: https://github.com/voxpupuli/puppet-systemd/blob/58f0075f6926e3efec647f1f4d525e9ed1ad006a/lib/facter/systemd.rb#L41

Reply:
Oh!  I opened #340 which should fix it.